### ISSUE TITLE
interaction.editReply is not a function

### ISSUE THREAD

**Issue by WannaBeChay** [ryzyx/discordjs-button-pagination#34]
![image](https://user-images.githubusercontent.com/54155663/146126565-1619ed6e-bdab-49c4-92b2-a218557e6558.png)

**Comment by WannaBeChay** [comment]
i have tryed to fix no nothing

**Comment by Mystic135** [comment]
ohhhh the same thing i am also facing
btw you tried to add more navigation buttons right?
Also i use replit yea

**Comment by Fab-is-insane** [comment]
Same issue with me

**Comment by ryzyx** [comment]
install latest version

**Comment by Mystic135** [comment]
> install latest version

does the latest version has a fix of this `interaction.editReply` error and has been made able to add more buttons rather than before or it showz still the same error and we cant add more buttons? And also why so late to reply to this?

**Comment by ryzyx** [comment]
yes it does for the error but more buttons cannot be added as not everyone wants multiple buttons for multiple functions. think of this as a basic library that helps u save time writing code for basic pagination. As for the late reply, sorry man am busy with studies

**Comment by Fab-is-insane** [comment]
Same error even after installing the latest version

**Comment by ryzyx** [comment]
Try now

**Comment by Mystic135** [comment]
> yes it does for the error but more buttons cannot be added as not everyone wants multiple buttons for multiple functions. think of this as a basic library that helps u save time writing code for basic pagination. As for the late reply, sorry man am busy with studies


Yes i understand that, but more buttons means that like u go to the first and the last page of the embed by clicking one button instead of clicking a button multiple times to go to either the first or the last page, as shown in the screenshot below.

![Screenshot_20220202-234015](https://user-images.githubusercontent.com/82505622/152207684-f99fa584-5a52-4ca8-adbc-3548e9bcb947.png)

**Comment by ryzyx** [comment]
yeah but not everyone needs that

**Comment by Mystic135** [comment]
> Try now

**ERROR:** `MessageEmbed.setFooter text must be a string` after downloading the new version

![Screenshot_20220206-095446](https://user-images.githubusercontent.com/82505622/152667192-601f5434-a5fd-496a-a0f1-fa091c4275e1.png)

![Screenshot_20220206-095333](https://user-images.githubusercontent.com/82505622/152667179-66a0d468-4f9a-4557-9c2f-3f8fe04f6637.png)

**Comment by dotnize** [comment]
> > Try now
> 
> **ERROR:** `MessageEmbed.setFooter text must be a string` after downloading the new version

What version of discord.js are you using? Have you tried updating to the latest stable version (v13.6.0)?

**Comment by Mystic135** [comment]
> > > Try now
> > 
> > 
> > **ERROR:** `MessageEmbed.setFooter text must be a string` after downloading the new version
> 
> What version of discord.js are you using? Have you tried updating to the latest stable version (v13.6.0)?

Yes, i updated my discord.js to `v13.6.0` from `v13.3.1`, and after that i reinstalled the latest version of this package, that old `interaction.editReply is not a function` error again.

![Screenshot_20220208-083626](https://user-images.githubusercontent.com/82505622/152907665-ae70446a-9dd1-4f7d-839c-fb1e2be82bea.png)

**Comment by dotnize** [comment]
This error can also occur when you're trying to pass a `Message` object (or any other object) instead of an interaction.
Are you sure you're using this package with an interaction (e.g, a slash command)?

If not, use the msg version instead
```
npm install discordjs-button-pagination@msg
```

**Comment by Mystic135** [comment]
> This error can also occur when you're trying to pass a `Message` object (or any other object) instead of an interaction. Are you sure you're using this package with an interaction (e.g, a slash command)?
> 
> If not, use the msg version instead
> 
> ```
> npm install discordjs-button-pagination@msg
> ```

Yes it finally worked, tysvm. I literally gave up for this package and was looking for some other packages, but with this

`npm i discordjs-button-pagination@msg`


and its functioning made me come back to use this package. Thx for the help, although had to wait about 2 months for the `interaction.editReply is not a function` error to be solved, but since its solved finally, thx a lot for the help and also the creation of this package 👌

**Comment by ofheather** [comment]
Hello, Sorry to meddle and not talk about it, but: You may be able to give me the link to download the background you have in the image? thank you very much in advance @WannaBeChay
